modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -398,10 +398,17 @@ export class TssUtils extends MpcUtils {
    * @param {SignatureShareRecord} signatureShare - a Signature Share
    * @returns {Promise<SignatureShareRecord>} - a Signature Share
    */
-  async sendSignatureShare(txRequestId: string, signatureShare: SignatureShareRecord): Promise<SignatureShareRecord> {
+  async sendSignatureShare(
+    txRequestId: string,
+    signatureShare: SignatureShareRecord,
+    path?: string
```

Comment:
add jsdoc for this param.

if this is undefined, do we assume that this is for the root wallet?

Reply:
Yes, if it's undefined, it's the root wallet. Will add some comments my morning tomorrow.

Reply:
Fixed